canonical/anbox-cloud.io
# Add global navigation

## Done

Adds global nav to the site.

## QA
- wait for the demo to come up and check that the global nav shows up.

Fixes #24
